kuzzleio/kuzzle-admin-console
# Fix #742 Generate raw filter from basic

Fixes #742 by adding a "Generate Raw" button to the Basic filter (Advanced Search in the UI).

### How should this be manually tested?

1. Create a Basic filter in the "Advanced Search" tab
2. Click the "Generate Raw" button
3. Check that the generated Raw filter is OK

### Other changes

A little bit of refactoring and code cleanup in the Filters.
